modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_identity/v1/IamIdentityExamples.java

```diff
@@ -145,11 +154,8 @@ public static void main(String[] args) throws Exception {
 
             // begin-create_api_key
 
-            CreateApiKeyOptions createApiKeyOptions = new CreateApiKeyOptions.Builder()
-                    .name(apiKeyName)
-                    .iamId(iamId)
-                    .description("Example ApiKey")
-                    .build();
+            CreateApiKeyOptions createApiKeyOptions = new CreateApiKeyOptions.Builder().name(apiKeyName).iamId(iamId)
```

Comment:
It looks like the code was re-formatted.  Note that the code fragments between the "begin" and "end" comments are extracted and used in the generation of the online API Reference Docs for the service.  So, make sure the code is formatted the way you want to see it in the generated docs.
Reference:
https://cloud.ibm.com/apidocs/iam-identity-token-api?code=java#create-api-key